# the configure script doesn't know that icc is equivalent
# with gcc
--- configure.orig	2012-05-02 06:17:08.000000000 +0200
+++ configure	2015-02-26 21:46:25.275559445 +0100
@@ -174,6 +174,7 @@
 case "$cc" in
   *gcc*) gcc=1 ;;
   *clang*) gcc=1 ;;
+  *icc*) gcc=1 ;;
 esac
 case `$cc -v 2>&1` in
   *gcc*) gcc=1 ;;
